PDSA_System/Server/Controllers/BrukerController.cs

```diff
@@ -94,7 +97,7 @@ public async Task<ActionResult<List<Bruker>>> UpdateBruker(Bruker bruker)
             using var conn = new DbHelper(connString).Connection;
 
             await conn.ExecuteAsync(
-                "UPDATE Bruker SET Fornavn = @Fornavn, Etternavn = @Etternavn, Email = @Email, PassordHash = @PassordHash, LederId = @LederId WHERE AnsattNr = @AnsattNr",
+                $"UPDATE Bruker SET Fornavn = @Fornavn, Etternavn = @Etternavn, Email = @Email, PassordHash = @PassordHash, LederId = @LederId WHERE AnsattNr = @AnsattNr",
```

Comment:
`PUT` overskriver passordhashes, men den skal jo ikke skrives på den måten uansett, da vi har en funksjon for å bytte passord
https://github.com/IS-200-G7/nordicdoor/blob/d119b3e3cbebf0e2f0bdfa67dd93bbaba472afa5/PDSA_System/Server/Controllers/BrukerController.cs#L135-L136
```suggestion
                $"UPDATE Bruker SET Fornavn = @Fornavn, Etternavn = @Etternavn, Email = @Email, LederId = @LederId WHERE AnsattNr = @AnsattNr",
```